### PR TITLE
LPS-146420 Update Selected Quote Page Layout to Mobile and Tablet View

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
Just starting reviewing :)

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/apps/selected-quote/components/quote-info.scss [hunk 1]
```diff
@@ -9,4 +9,8 @@
 			padding-top: 0 !important;
 		}
 	}
+
+	@media screen and (max-width: 991px) {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/apps/selected-quote/components/quote-info.scss:13]: Could you use the tablet variable ```$breakpoint-tablet``` in the ```variables.scss```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@media screen and (max-width: 991px) {
+  .selected-quote-header h1 {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.css:2]: Could you use ```dialect``` and ```bootstrap``` for that?

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    font-size: 32px;
+  }
+  
+  .selected-quote-header h3 {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.css:6]: Could you use ```dialect``` and ```bootstrap``` for that?

**Review by silasbsi — CHANGES_REQUESTED** [review]
There are some adjustments you need to make.

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
@kevenleone could you make a review too? :fire:

**Review by kevenleone — APPROVED** [review]
PR looks good, congratulations @JoMaAlves

🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**